src/controllers/applicationsContentController.js

```diff
@@ -0,0 +1,304 @@
+const { renderTemplate } = require('../utils/util');
+const config = require(process.cwd() + '/config');
+const cpToken = require(process.cwd() + '/cpToken');
+const constants = require('../utils/constants');
+const path = require('path');
+const fs = require('fs');
+const axios = require('axios');
+const https = require('https');
+
+const filePrefix = config.pathToContent;
+const certPath = path.join(process.cwd(), config.certificate.path);
+const certPassword = config.certificate.password;
+const controlPlaneUrl = config.controlPlaneUrl;
+const token = cpToken.token;
+
+const httpsAgent = new https.Agent({
+    ca: fs.readFileSync(certPath),  
+    rejectUnauthorized: true,
+});
+
+// ***** Load Applications *****
+
+const loadApplications = async (req, res) => {
+    const orgName = req.params.orgName;
+    let html, metaData, templateContent;
```

Comment:
lets use const where possible

Reply:
Those variables are not constants and values get assigned to them later